### ISSUE TITLE
Global Admin Breadcrumb On Agency Dashboard

### ISSUE THREAD

**Issue by o-fish-wildaid** [WildAid/o-fish-web#208]
When a global admin clicks on Dashboards -> Agencies and then on an agency's name - it takes them to the agency dashboard (with compliance rate etc.). 

On that page - /agency_charts/AgencyName - just above the agency name should be breadcrumbs.

The text 'Dashboard/Agencies' - which links back to the Agencies Dashboard (same page as going to Dashboard->Agencies). 

Design as per https://mongodb.invisionapp.com/share/MYXRXC2T2H3#/422348696_Global_Admin-Dashboard-Agency

The current page looks like this, without a breadcrumb.
![Screen Shot 2020-10-02 at 1 16 54 PM](https://user-images.githubusercontent.com/1767528/94951018-8c082a80-04b1-11eb-8a18-999aef4ce389.png)

**Comment by SEGH** [comment]
Hi! Thanks for the great kickoff event, @Sheeri! I'm new to contributing to open source and am interested in taking on this issue

**Comment by Sheeri** [comment]
Hi @SEGH ! It was great to have you on the call. I just realized that I labeled this issue incorrectly - it's global admin, so I added the 'needs own instance' issue - I've assigned you the issue. If you don't want to build your own foundation now, that's understandable, just let me know and I can unassign you.

Otherwise, the build docs are at https://wildaid.github.io/build/ and I'm happy to help with any questions you have!
